src/test/regress/expected/failure_copy_to_reference.out

```diff
@@ -0,0 +1,434 @@
+-- 
+--  Failure tests for COPY to reference tables 
+-- 
+CREATE SCHEMA copy_reference_failure;
+SET search_path TO 'copy_reference_failure';
+-- we don't want to see the prepared transaction numbers in the warnings
+SET client_min_messages TO ERROR;
+SELECT citus.mitmproxy('conn.allow()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+-- With one placement COPY should error out and placement should stay healthy.
+SET citus.shard_replication_factor TO 1;
```

Comment:
Minor: setting the `shard_replication_factor ` and `shard_count ` is slightly confusing since we don't rely on those